components/script/dom/htmlimageelement.rs

```diff
@@ -152,10 +152,16 @@ impl FetchResponseListener for ImageContext {
             self.id,
             FetchResponseMsg::ProcessResponse(metadata.clone()));
 
-        let metadata = metadata.ok().map(|meta| {
-            match meta {
-                FetchMetadata::Unfiltered(m) => m,
-                FetchMetadata::Filtered { unsafe_, .. } => unsafe_
+        // TODO how to get to document?
+        // is this the right spot to process the network timing metadata
```

Comment:
process_response_eof probably makes more sense, since that's called after the response is complete.